apps/etterlatte-saksbehandling-ui/client/src/components/person/journalfoeringsoppgave/journalpost/modal/SoekPersonPdl.tsx

```diff
@@ -9,12 +9,12 @@ import Spinner from '~shared/Spinner'
 import { ApiErrorAlert } from '~ErrorBoundary'
 import { Adressevisning } from '~components/behandling/felles/Adressevisning'
 import { formaterKanskjeNavn } from '~components/behandling/soeknadsoversikt/familieforhold/SamsvarPersongalleri'
-import { Bruker, BrukerIdType } from '~shared/types/Journalpost'
+import { BrukerIdType, SoekPersonVelg } from '~shared/types/Journalpost'
 
 interface SoekPersonProps {
   open: boolean
   setOpen: Dispatch<SetStateAction<boolean>>
-  oppdaterBruker: (bruker: Bruker) => void
+  oppdaterBruker: (person: SoekPersonVelg) => void
```

Comment:
Siden dette skal brukes flere steder ville jeg kanskje døpt om litt her. Du oppdaterer ikke nødvendigvis bruker. 

Noe i gaten `velgPerson` gir for meg mer mening, og koden som drar inn `SoekPersonPdl(Modal)`tar stilling til hva de gjør ved handling "person valgt"

Reply:
https://github.com/navikt/pensjon-etterlatte-saksbehandling/pull/5163/commits/fad7a9a25d0310ca5e08d0079453b6e5f5a0e240